TF_Settings_Web/package.json

```diff
@@ -10,8 +10,11 @@
         "react": "^17.0.2",
         "react-canvas-confetti": "^1.3.0",
         "react-circular-progressbar": "^2.1.0",
+        "react-colorful": "^5.6.1",
         "react-dom": "^17.0.2",
+        "rgb-hex": "^4.0.0",
```

Comment:
🙈 This doesn't seem to be used anywhere, can we remove?

Reply:
Good catch - was let in from trying out a different solution